front/app/components/ErrorToast/index.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+// components
+import {
+  Box,
+  Icon,
+  colors,
+  Text,
+  useBreakpoint,
+} from '@citizenlab/cl2-component-library';
+import CloseIconButton from 'components/UI/CloseIconButton';
+
+// utils
+import styled from 'styled-components';
+import eventEmitter from 'utils/eventEmitter';
+
+// events
+import { BUDGET_EXCEEDED_ERROR_EVENT } from 'components/ParticipationCTABars/VotingCTABar/events';
+import {
+  VOTES_EXCEEDED_ERROR_EVENT,
+  VOTES_PER_OPTION_EXCEEDED_ERROR_EVENT,
+} from 'components/AssignMultipleVotesControl';
+
+// intl
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+const StyledBox = styled(Box)`
+  opacity: 0;
+  z-index: 1000;
+  transition: all 0.6s ease-in-out;
+
+  &.visible {
+    opacity: 1;
+  }
+`;
+
+const ErrorToast = () => {
+  const atPageEnd = useRef(false);
+  const [error, setError] = useState<string | null>(null);
+  const [showError, setShowError] = useState(false);
+  const { formatMessage } = useIntl();
+  const [marginBottom, setMarginBottom] = useState('0px');
+  const isMobileView = useBreakpoint('phone');
+
+  // Listen for budgeting exceeded error
+  useEffect(() => {
+    const subscription = eventEmitter
```

Comment:
Not sure if there is a better way to listen for all errors in a single useEffect? :thinking: